src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -305,21 +315,23 @@ bool TuneParamsSelector::VerifyTuneParams(const fully_connected_params& params,
     if (tparams.tile_ofm * simd > 64)
         return false;
 
-    bool is_i4_u4 = (params.weights.GetDType() == WeightsType::INT4 || params.weights.GetDType() == WeightsType::UINT4);
+    bool is_i4_u4_u8 = (params.weights.GetDType() == WeightsType::INT4 || params.weights.GetDType() == WeightsType::UINT4 ||
+                        params.weights.GetDType() == WeightsType::UINT8);
     if (tparams.kernel_type == FullyConnected_bf_tiled::KernelType::SLM) {
         const auto required_batch_alignment = 64;
         if (!params.is_shape_agnostic && (!IsAligned(output_b, required_batch_alignment) || output_b < min_slm_size))
             return false;
 
         const auto required_tile_b = 8;
-        if ((tparams.tile_b != required_tile_b) && !is_i4_u4)
+        if ((tparams.tile_b != required_tile_b) && !is_i4_u4_u8)
             return false;
 
         const auto required_tile_ofm = 2;
         if (tparams.tile_ofm != required_tile_ofm)
             return false;
 
-        if (params.weights.GetDType() != WeightsType::INT4 && params.weights.GetDType() != WeightsType::UINT4)
+        if (params.weights.GetDType() != WeightsType::INT4 && params.weights.GetDType() != WeightsType::UINT4 &&
+            params.weights.GetDType() != WeightsType::UINT8)
```

Comment:
nit: `if (!is_i4_u4_u8)`?

Reply:
I added is_weight_dyn_quantizable function to make this line simple.